cmd/juju/machine/upgradeseries_test.go

```diff
@@ -36,7 +38,13 @@ func (s *UpgradeSeriesSuite) runUpgradeSeriesCommandWithConfirmation(c *gc.C, co
 	ctx.Stdin = &stdin
 	stdin.WriteString(confirmation)
 
-	com := machine.NewUpgradeSeriesCommandForTest()
+	// mock remote API
+	mockController := gomock.NewController(c)
+	mockUpgradeSeriesAPI := mocks.NewMockUpgradeMachineSeriesAPI(mockController)
+	mockUpgradeSeriesAPI.EXPECT().UpgradeSeriesPrepare(gomock.Any()).AnyTimes()
```

Comment:
assuming we'll change this later?

Reply:
@hmlanigan here the `AnyTimes(gmock.Any())` suggests that the method is disinterested in the number of times the API is called or what argument is passed to it. In other words the code says "Just pretend the external dependency is doing the correct thing". We are not testing to see if the external dependency is being called the correct number of times.

Reply:
I agree for the tests of the cli args, but why wouldn't we want to check it for TestCompleteCommand()?  To to test that a failure in machine.UpgradeSeriesPrepare is handled correctly?